run-devel.sh

```diff
@@ -58,6 +58,8 @@ if [ ! "$(docker ps -q -f name=$CONTAINER)" ]; then
             --volume="/etc/sudoers.d:/etc/sudoers.d:ro" \
             --volume="/tmp/.X11-unix:/tmp/.X11-unix:rw" \
             --privileged \
+            --gpus all \
```

Comment:
Is a privileged container necessary for using gpu?

Reply:
I don't know. However, the privileged is useful for other things, like accessing USB devices.

Reply:
> I don't know. However, the privileged is useful for other things, like accessing USB devices.

Got it, but maybe we should keep the minimum requirement. I didn't look deep, but apparently running privileged container is a [bad idea](https://www.trendmicro.com/en_us/research/19/l/why-running-a-privileged-container-in-docker-is-a-bad-idea.html). We can always set the specifics capabilities that we are in need, like [here](https://stackoverflow.com/a/36441605)

Reply:
We can remove it but I would consider adding a parameter to the script to pass extra arguments

Reply:
I agree, for development, privileged will make our life easier during development, however, it is not safe